R/vault.R

```diff
@@ -0,0 +1,4 @@
+vault_read <- function(key, field) {
+  vault <- vaultr::vault_client(login = "github")
```

Comment:
vaultr now reliably caches github tokens so there's no need to do anything clever here with the caching layer that was in `zzz.R`